cts.schema.json

```diff
@@ -112,7 +122,19 @@
       "type": "boolean",
       "const": true,
       "description": "The flag indicating that the selector is not a valid JSONPath selector expression"
+    },
+    "result_paths": {
+      "description": "The expected normalized JSONPath query strings, one for each result value",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "results_paths": {
```

Comment:
Ordinarily, I'd consider this an unnecessary refactor. Since the definition is only used in one place, it makes sense to just inline it. 

But we're doing this for other definitions, so... okay.

Reply:
Hi @greg, I assumed it was done this way so `test_case_result` (and now `test_case_results`) could be referenced remotely by URL form another schema. Happy to change it if you'd prefer it inline.

Reply:
It's fine as is